merkle/index.go

```diff
@@ -0,0 +1,78 @@
+package merkle
+
+import "sort"
+
+// siblingIndecies returns indecirs of sibling elements
```

Comment:
```suggestion
// siblingIndecies returns indecies of sibling elements
```

Reply:
fixed in [this](https://github.com/ComposableFi/go-merkle-trees/pull/22/commits/2e360cdbbfc6c3e12c3c206b0c3b3d92e91c702c)